packages/calcite-components/src/components/tile-group/tile-group.tsx

```diff
@@ -52,16 +102,77 @@ export class TileGroup implements InteractiveComponent {
 
   @Element() el: HTMLCalciteTileGroupElement;
 
+  private items: HTMLCalciteTileElement[] = [];
+
+  private slotEl: HTMLSlotElement;
+
   //--------------------------------------------------------------------------
   //
   //  Private Methods
   //
   //--------------------------------------------------------------------------
 
-  private updateTiles = (): void => {
-    this.el.querySelectorAll("calcite-tile").forEach((item) => (item.scale = this.scale));
+  private mutationObserver = createObserver("mutation", () => this.updateItems());
+
+  private setSelectedItems = (emit: boolean, elToMatch?: HTMLCalciteTileElement): void => {
```

Comment:
For consistency, can you set a default to `emit` and make it the last argument? This will align with other methods that conditionally emit.

Reply:
I refactored the emit argument away entirely, much simpler now.